.github/workflows/unit-tests.yml

```diff
@@ -10,7 +10,6 @@ jobs:
           - { make: 'check-memory', cc: 'gcc', tag: 'latest' }
           - { make: 'distcheck', cc: 'clang', tag: 'latest' }
           - { make: 'distcheck', cc: 'gcc', tag: 'i386' }
-          - { make: 'pycheck', cc: 'gcc', tag: 'latest' }
```

Comment:
I was going to complain "unit-tests-refresh.yml needs the same change", but turns out we never added it there in the first place. It should call pytest-cov, however (separate commit/PR).